gc/base/standard/Scavenger.hpp

```diff
@@ -70,6 +70,15 @@ class MM_Scavenger : public MM_Collector
 	/*
 	 * Data members
 	 */
+public:
+	struct CycleTimes {
+		/* The following start/end times record total cycle duration, done only by main thread. */
```

Comment:
Edit to include "cycle increment durations"... "total cycle and cycle increment durations"